### DISCUSSION TITLE
Netflix Help - SMPTE base

### DISCUSSION THREAD

**Discussion by uckthis** [SubtitleEdit/subtitleedit#5083] [Q&A]
I got a pain-in-the-ass client who's doing my head in.  First he said that there shouldn't be any 3-11 gaps between subtitles, which I fixed and SE Netflix Quality check doesn't show any errors. But he still says there are a few 3-11 gaps. And now he's saying it's because I export the file in ms and not frames. According to him, ttp:timeBase="media" should be ttp:timeBase="SMPTE". DFXP created with SE doesn't have this string, the STMPE-TT 2052.xml has a ttp:timeBase="media" and no SMPTE. I have Caption Maker and Ooona and those two also do the ttp:timeBase="media" and  not SMPTE. 
He says it should look like this:

![image001](https://user-images.githubusercontent.com/28671050/120371634-85c39a00-c32f-11eb-881e-6e62b9b2987e.png)

Any ideas?

**Comment by niksedk** [comment]
What subtitle file format are you saving in?
If you're using "Timed Text 1.0", try File - Timed Text properties... and change "Time base":
![image](https://user-images.githubusercontent.com/1261331/120428502-0b693900-c374-11eb-841b-27ed3f823a6e.png)

**Comment by uckthis** [comment]
@niksedk  I was using Netflix DFXP. Anyway, I figured it out in Ooona and exported it as a DFXP with SMPTE.
Thank you.
